product_docs/docs/biganimal/release/using_cluster/04_backup_and_restore.mdx

```diff
@@ -2,11 +2,20 @@
 title: "Backing up and restoring"
 ---
 
-### Backups
+### Backups 
+ 
+BigAnimal backs up the data in your PostgreSQL clusters. Depending on which cloud your cluster is deployed, BigAnimal uses either of the following cloud object storage solutions:
 
-BigAnimal backs up the data in your PostgreSQL clusters. Backups are stored in Azure Blob Storage, in the "hot" access tier with geo-zone-redundant storage (GZRS). You are responsible for the charges associated with backup storage. See [Azure Blob Storage documentation](https://docs.microsoft.com/en-us/azure/storage/blobs/) for more information.
+| Cloud | Object storage solution                                                        |   |
+|----------------------------|--------------------------------------------------------------------------------------|---|
+| AWS                      | Amazon S3  (standard tier)                                                                          |   |
```

Comment:
We should indicate somewhere that AWS backups are *not* cross-regionally durable (yet).

Reply:
Added a note to the documentation: 

!!! Note
    Backups in AWS do not currently support cross-regional replication.